packages/react-native-win32-tester/src/js/components/RNTesterEmptyBookmarksState.win32.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+import * as React from 'react';
+import {View, Image, Text, StyleSheet} from 'react-native';
+
+export const RNTesterEmptyBookmarksState = (): React.Node => (
+  <View style={styles.emptyContainer}>
+    <View style={styles.emptyContainerInner}>
+      <Image
+        source={require('../assets/empty.png')}
+        resizeMode="contain"
+        style={styles.emptyImage}
+      />
+      <View>
+        <Text style={styles.heading}>Bookmarks are empty</Text>
+        <Text style={styles.subheading}>
+          Please tap the{' '}
+          {/* [Win32] remove the image since nested non-Text in text is unsupported in NetUI*/}
```

Comment:
Suggestion: Maybe we have a emoji or other unicode char that matches the image?

Reply:
Yeah, I want to change this to something decent upstream (tracking with https://github.com/microsoft/react-native-windows/issues/6341). ACoates had the text "Bookmark" in the XAML version previously, but I think we can also accomplish the near-exact representation that's wanted using Flexbox with overflow.